docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '3.4'
+
+services:
+  web:
+    build:
+      context: .
```

Comment:
You only need to specify build context or dockerfile if you want something different from the _default_. Looks like all defaults here.